bin/autosign

```diff
@@ -52,8 +52,9 @@ command :generate do |c|
 
   c.action do |global_options,options,args|
     config = Autosign::Config.new({'config_file' => global_options['config']})
-    global_options['secret'] = config.settings['jwt_token']['secret'] if global_options['secret'].nil?
-    options['validfor'] = config.settings.to_hash['jwt_token']['validity'].to_s if options['validfor'] == '7200'
+    config_settings = config.settings
```

Comment:
Why the local variables here?

Reply:
config.settings doesn't cache the settings so every time config.settings is called it reads the settings file all over again, logs things, merges things.  This is the least I could do to "cache" the settings without an overhaul to the config class.  If I overhauled it I would just use a singleton class for config.   But that would be a major change IMHO.